test/test_ops.py

```diff
@@ -227,17 +227,26 @@ def _ref_test_helper(self, ctx, device, dtype, op, skip_zero_numel=False, skip_z
                 )
             ):
                 continue
+
+            # Note: this deliberately uses 'op.torch_opinfo' in both cases
+            # because 'op.aliases' is empty.  When the aliases are executed
+            # within the context, they will behave as refs.  This is done to
+            # check for compatibility between all APIs.
+            aliases = op.torch_opinfo.aliases if op.test_aliases else ()
             with ctx():
-                ref_result = op(sample.input, *sample.args, **sample.kwargs)
-            torch_result = op.torch_opinfo(sample.input, *sample.args, **sample.kwargs)
+                ref_results = [f(sample.input, *sample.args, **sample.kwargs)
```

Comment:
I think it'd be clearer to enumerate in a loop outside and do each comparison directly vs. gathering the comparisons. It would also make it easier to provide an error message in the future, because the error would be thrown within the loop for a particular alias